BackendTaskPilot/BackendTaskPilot.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
```

Comment:
Is there any particular reason we're going for net7.0? Version 8 was released almost a year ago. Possibly we could go with the RC 2 for v9 depending on when we plan to release, as v9.0 is supposed to be released in November.